cmd/dkvsrv/main.go

```diff
@@ -81,8 +85,22 @@ func main() {
 	setFlagsForNexusDirs()
 	setupStats()
 
+	var secure = false
+	if caCertPath != "" && keyPath != "" && certPath != "" {
+		secure = true
```

Comment:
Please get rid of the `if` block and directly assign the conditional expression to the `secure` variable.